src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -154,4 +166,104 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
             }
         }));
     }
+
+    private async Task SubscribeDashboardLogsAsync()
+    {
+        string? dashboardResourceId = null;
+
+        try
+        {
+            using var cts = new CancellationTokenSource(TimeSpan.FromMinutes(1));
+            await foreach (var @event in resourceNotificationService.WatchAsync(cts.Token))
```

Comment:
```suggestion
            await foreach (var notification in resourceNotificationService.WatchAsync(cts.Token))
```

So you don't have to use `@event` everywhere.

Reply:
Done